addOns/graphql/src/test/java/org/zaproxy/addon/graphql/GraphQlFingerprinterUnitTest.java

```diff
@@ -309,21 +309,28 @@ private static String errorResponse(String error, String field, boolean data) {
                 + " }";
     }
 
+    @SuppressWarnings("null")
     @ParameterizedTest
     @MethodSource("fingerprintData")
     void shouldFingerprintValidData(String graphqlImpl, String response) throws Exception {
         // Given
-        ExtensionAlert extensionAlert = mockExtensionAlert();
         nano.addHandler(new GraphQlResponseHandler(response));
         var fp = new GraphQlFingerprinter(UrlBuilder.build(endpointUrl));
+        DiscoveredGraphQlEngine[] discoveredEngine = new DiscoveredGraphQlEngine[1];
```

Comment:
Can we use a `java.util.List` and append to it instead?

Reply:
Because of the lambda it hits the must be final error/warning.